EDEngineer/Resources/Data/entryData.json

```diff
@@ -1874,4 +1874,2476 @@
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Tungsten Carbide",
+    "ValueCr": null,
+    "BarterCost": 11,
+    "BarterValue": 6,
+    "FormattedName": "tungstencarbide",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": ["LAB", "PROC"],
+    "ContainerType": ["Industrial Locker (S)", "Industrial Locker (L)", "Research Locker (L)"]
+    },
+    {
+    "Name": "Viscoelastic Polymer",
+    "ValueCr": 600,
+    "BarterCost": 11,
+    "BarterValue": 6,
+    "FormattedName": "viscoelasticpolymer",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Chemicals",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Weapon Component",
+    "ValueCr": 1000,
+    "BarterCost": 18,
+    "BarterValue": 9,
+    "FormattedName": "weaponcomponent",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Accident Logs",
+    "ValueCr": 4000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "accidentlogs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": ["LAB", "PROC", "PWR", "AGRI"],
+    "ContainerType": ["Agricultural Data Port", "Data Port", "Industrial Data Port", "Laboratory Data Port", "Power Data Port"]
+    },
+    {
+    "Name": "Air Quality Reports",
+    "ValueCr": 3000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "airqualityreports",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Industrial Data Port", "Power Data Port"]
+    },
+    {
+    "Name": "Atmospheric Data",
+    "ValueCr": 4000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "atmosphericdata",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Laboratory Data Port"]
+    },
+    {
+    "Name": "Audio Logs",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "audiologs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Security Data Port"]
+    },
+    {
+    "Name": "AX Combat Logs",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,

```

Comment:
all the `null` fields can be completely removed

```json
{
   "Value": 10,
   "Something": null
}```

is equivalent to

```json
{
   "Value": 10
}```

Reply:
In a few cases the null value indicates that we are missing the value, as in we dont know it yet.
I will leave those inn, or should I put in a diffrent placeholder?

Reply:
you could put -1 on the ones you think are missing - it will be a reminder of incomplete data in the gui so it's not forgotten

Reply:
Ok, the missing data is the ValueCr for some items.